roles/init_dbserver/tasks/verify_tde_vars.yml

```diff
@@ -1,7 +1,7 @@
 ---
-- name: Verify edb_key_wrap_cmd and edb_key_unwrap_cmd
+- name: Verify edb_key_wrap_cmd and edb_key_unwrap_cmd are defined
   ansible.builtin.fail:
-    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster with TDE"
+    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster"
```

Comment:
should include something about `with TDE`, these are not required for initializing a cluster, but are for initializing a cluster with TDE.